include/lsst/coadd/utils/copyGoodPixels.h

```diff
@@ -71,6 +71,27 @@ namespace utils {
         lsst::afw::image::MaskPixel const badPixelMask ///< skip input pixel if src mask & badPixelMask != 0
     );
 
+    /**
+    * @brief copy good pixels from one covariance image to another
+    *
+    * Good pixels are those for which mask & badPixelMask == 0.
+    *
+    * Only the overlapping pixels (relative to the parent) are copied;
+    * thus the images do not have to be the same size.
```

Comment:
Perhaps "destination" rather than "parent" here, since that's the term you're using elsewhere.